components/base/Collapsible.tsx

```diff
@@ -21,7 +29,7 @@ const CollapsibleWrapper = styled.div`
     font-size: 18px;
     line-height: 27px;
     :hover {
-      box-shadow: ${BOX_SHADOW_7};
+      box-shadow: ${BOX_SHADOW_4};
```

Comment:
Changed the Hover state to a different Box-shadow to somehow differentiate it from a focus state. Still not sure this works well. Open to any ideas and thoughts!

Reply:
I dig it! It's subtle, but I think it works!